tf2onnx/utils.py

```diff
@@ -146,22 +148,40 @@ def get_tf_tensor_data(tensor):
     return np_data
 
 
-def get_shape(node):
+def get_tf_const_value(op, as_list=True):
+    make_sure(is_const_op(op), "{} isn't a const op".format(op.name))
+    value = get_tf_tensor_data(op.get_attr("value"))
+    if as_list:
+        value = value.tolist()
+    return value
+
+
+def get_tf_shape(node):
```

Comment:
maybe it is better to not handle const and shape attr together.
maybe a ```get_tf_shape_attr``` and ```get_tf_const_shape``` (or just get shape from ```get_tf_const_value```)?

Reply:
get const shape isn't used, just handle the attr case now.